### DISCUSSION TITLE
应用IPAdapterFaceID一直报错

### DISCUSSION THREAD

**Discussion by xiaoxiao78217821** [cubiq/ComfyUI_IPAdapter_plus#762] [General]
应用IPAdapterFaceID
Error(s) in loading state_dict for ProjModelFaceIdPlus:
size mismatch for perceiver_resampler.proj_in.weight: copying a param with shape torch.Size([768, 1280]) from checkpoint, the shape in current model is torch.Size([768, 1664]).

**Comment by GrachYAN** [comment]
你可以试试改用vit-h格式的clip_vision模型
